pkg/controller/render/render_controller.go

```diff
@@ -636,3 +763,447 @@ func getMachineConfigsForPool(pool *mcfgv1.MachineConfigPool, configs []*mcfgv1.
 	}
 	return out, nil
 }
+
+func (ctrl *Controller) experimentalAddImageStreams(pool *mcfgv1.MachineConfigPool) error {
+
+	var err error
+
+	// Make sure we have an image stream to render content into
+	renderedImageStreamName := pool.Name + ctrlcommon.ImageStreamSuffixRenderedConfig
+
+	// And also, an imagestream to receive our "coreos-derive" build image
+	contentImageStreamName := pool.Name + ctrlcommon.ImageStreamSuffixMcoContent
+
+	// And allow for the user to specify some stuff afterwards
+	customImageStreamName := pool.Name + ctrlcommon.ImageStreamSuffixMcoContentCustom
+
+	// And then if we're feeling dangerous, allow for an external image
+	externalImageStreamName := pool.Name + ctrlcommon.ImageStreamSuffixMcoContentExternal
+
+	// Our list of imagestreams we need to ensure exists
+	var ensureImageStreams = []string{
+		renderedImageStreamName,
+		contentImageStreamName,
+		customImageStreamName,
+		externalImageStreamName,
```

Comment:
I'm uncertain whether we should auto-create the external stream.  But, fine for now!